### PR TITLE
LPD-35640 - Change Treeview modal size

### PR DESCRIPTION
**Pull request by juanjofgliferay**
## Story / Task
[LPD-26397](https://liferay.atlassian.net/browse/LPD-26397) / [LPD-35640](https://liferay.atlassian.net/browse/LPD-35640)

## Goal
Actual size of the modal that contains the TreeView for field selection is too wide and header and footer content usually does not align correctly with the body content.
To prevent this behavior, changing the size of the modal to `lg` and applying a [`modal-height-full` Clay Modal utility class](https://clayui.com/docs/components/modal/markup.html#css-height-full-screen) make it possible to have a full height modal of any size.

## Before
image

## After
image

### CODE REVIEW

File: modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/filters/components/Configuration.tsx [hunk 1]
```diff
@@ -106,6 +106,7 @@ function Configuration({
 
 	const openSelectFieldModal = () => {
 		openModal({
+			className: 'modal-height-full',
```
Comment (markocikos) [inline modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/filters/components/Configuration.tsx:109]: Nice! Normal `lg` modal had many issues with overflow, did you check if both horizontal and vertical overflows behave as expected?

**Review by markocikos — COMMENTED** [review]
@juanjofgliferay A question inline

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_eu.properties [hunk 2]
```diff
@@ -14872,8 +14872,8 @@ release-feature-flag-configuration-name=Release Feature Flags (Automatic Copy)
 release-feature-flag[DISABLE_PRIVATE_LAYOUTS]=Disable Private Pages (Automatic Copy)
 release-hold=Release Hold (Automatic Copy)
 release-manager-configuration-name=Release Manager (Automatic Copy)
-relevance=Relevance (Automatic Copy)
-relevance-can-be-turned-on-or-off-but-not-removed=Relevance can be turned on or off, but not removed. (Automatic Copy)
+relevance=Adierazgarritasuna
+relevance-can-be-turned-on-or-off-but-not-removed=Adierazgarritasuna aktibatu ala desaktibatu daiteke, baina ezin da kendu.
```
[the remark this answers is not in the document]
Comment (juanjofgliferay) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_eu.properties:14876]: :man_shrugging:

File: modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/filters/components/Configuration.tsx (hunk 1 above)
Comment (juanjofgliferay) [reply modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/filters/components/Configuration.tsx:109]: I was concerned about those issues with `lg` but everything seems to be ok (smoke tests + playwright)

image

image

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_eu.properties (hunk 2 above)
[the remark this answers is not in the document]
Comment (markocikos) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_eu.properties:14876]: Looks like you jumped in the small window when our fork did not yet update :slightly_smiling_face:

**Review by markocikos — APPROVED** [review]
@juanjofgliferay LGTM! :sparkles:

**Comment by juanjofgliferay** [comment]
ci:forward

**Comment by juanjofgliferay** [comment]
ci:forward

**Pull request closed without merging**